vision/google/cloud/vision/_gax.py

```diff
@@ -25,10 +26,14 @@ class _GAPICVisionAPI(object):
 
     :type client: :class:`~google.cloud.vision.client.Client`
     :param client: Instance of ``Client`` object.
+
+    :type kwargs: dict
+    :param kwargs: Additional keyword arguments are sent to the GAPIC client.
```

Comment:
Updating argument and not docstring fail. I considered kwargs and then decided against it.